.github/workflows/pa11y-accessibility.yml

```diff
@@ -0,0 +1,58 @@
+name: pa11y-accessibility-check
+on:
+  pull_request:
+    branches: [main]
+jobs:
+  pa11y:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22.x
+      - name: Install dependencies
+        run: npm install
+      - name: Build the app
+        run: npm run build
+      - name: Install Pa11y
+        run: npm install -g pa11y
+      - name: Install Chromium dependencies (for headless env)
```

Comment:
Because pa11y runs against a website, we have to launch the app for pa11y to check. Chromium allows us to do this.